fix(modal): Modal open focus fix #343

airikej opened the pull request.

HarlesPilter commented:
Any idea why this changes fixes it and why it broke?
Also I can't find a link for the deployed storybook

airikej commented:
> Any idea why this changes fixes it and why it broke? Also I can't find a link for the deployed storybook

https://github.tehik.ee/tedi-design-system/fix/343-modal-modal-is-not-focused-when-opened/?path=/docs/community-modal--docs
No idea why it wasn't deployed before. The code i removed, I've added earlier to combat the ClosingButton focus "problem" - the ClosingButton was in focused state the moment modal was opened, but that ref apparently broke the modal focus itself
